src/components/CategoryController.jsx

```diff
@@ -22,7 +22,7 @@ export default class CategoryController extends FalcorController {
       const categoryData = this.state.data.categories[category];
       //console.log("Data: " + JSON.stringify(categoryData));
       return (
-        <div>
+        <div className="category">
```

Comment:
I'm not sure you can put className at the top level. This previously broke transitions, but I've also since changed transitions. Need to check if this still breaks. Will get back to you soon.